build-tools/src/main/kotlin/org/jetbrains/kotlin/Reporter.kt

```diff
@@ -74,12 +72,14 @@ open class Reporter : DefaultTask() {
             "${reportJson.statistics.report}\n ${reportEpilogue()}"
 
         project.logger.info(report)
-        sendTextToSlack(report)
+        if (doSlackSending())
+            sendTextToSlack(report)
     }
-
-
 }
 
+private fun DefaultTask.doSlackSending() = !project.hasProperty("build.reporter.noSlack")
+        || project.property("build.reporter.noSlack").toString().toBoolean()
```

Comment:
So if `-Pbuild.reporter.noSlack=true`, the script does send Slack report?

Reply:
oops